frontend/src/scenes/web-analytics/SessionAttributionExplorer/__mocks__/sessionAttributionQuery.json

```diff
@@ -13,7 +13,7 @@
     ],
     "error": null,
     "hasMore": false,
-    "hogql": "SELECT\n    count() AS `context.columns.count`,\n    $channel_type AS `context.columns.channel_type`,\n    $entry_referring_domain AS `context.columns.referring_domain`,\n    $entry_utm_source AS `context.columns.utm_source`,\n    $entry_utm_medium AS `context.columns.utm_medium`,\n    topK(10)($entry_utm_campaign) AS `context.columns.utm_campaign`,\n    topK(10)(nullIf(arrayStringConcat([if(isNotNull($entry_gclid), 'glcid', NULL), if(isNotNull($entry_gad_source), 'gad_source', NULL)], ','), '')) AS `context.columns.ad_ids`,\n    topK(10)($entry_current_url) AS `context.columns.example_entry_urls`\nFROM\n    sessions\nWHERE\n    and(1, 1, less($start_timestamp, toDateTime('2024-07-10 09:38:12.535618')), greaterOrEquals($start_timestamp, toDateTime('2024-07-03 09:38:12.535669')))\nGROUP BY\n    `context.columns.channel_type`,\n    `context.columns.referring_domain`,\n    `context.columns.utm_source`,\n    `context.columns.utm_medium`\nORDER BY\n    `context.columns.count` DESC\nLIMIT 101\nOFFSET 0",
+    "hogql": "SELECT\n    count() AS `context.columns.count`,\n    $channel_type AS `context.columns.channel_type`,\n    $entry_referring_domain AS `context.columns.referring_domain`,\n    $entry_utm_source AS `context.columns.utm_source`,\n    $entry_utm_medium AS `context.columns.utm_medium`,\n    topK(10)($entry_utm_campaign) AS `context.columns.utm_campaign`,\n    topK(10)(nullIf(arrayStringConcat([if(isNotNull($entry_gclid), 'gclid', NULL), if(isNotNull($entry_gad_source), 'gad_source', NULL)], ','), '')) AS `context.columns.ad_ids`,\n    topK(10)($entry_current_url) AS `context.columns.example_entry_urls`\nFROM\n    sessions\nWHERE\n    and(1, 1, less($start_timestamp, toDateTime('2024-07-10 09:38:12.535618')), greaterOrEquals($start_timestamp, toDateTime('2024-07-03 09:38:12.535669')))\nGROUP BY\n    `context.columns.channel_type`,\n    `context.columns.referring_domain`,\n    `context.columns.utm_source`,\n    `context.columns.utm_medium`\nORDER BY\n    `context.columns.count` DESC\nLIMIT 101\nOFFSET 0",
```

Comment:
Actually, this should include many more ad IDs, but I'm fine to keep this as is now